bokehjs/src/lib/core/enums.ts

```diff
@@ -80,6 +80,9 @@ export const ImageOrigin = Enum("bottom_left", "top_left", "bottom_right", "top_
 export type LatLon = "lat" | "lon"
 export const LatLon = Enum("lat", "lon")
 
+export type LegendBackgroundPolicy = typeof LegendBackgroundPolicy["__type__"]
+export const LegendBackgroundPolicy = Enum("none", "even", "odd", "every")
```

Comment:
This set of values seems like it could be useful in other contexts.  Can we give it a more general name to avoid forestalling future possibilities? e.g. `AlternationPolicy`? I could image application for grids, or various tables, etc. in the future.